src/UnsupportedFeatureException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace React\EventLoop;
+
+use Exception;
+
+final class UnsupportedFeatureException extends Exception
+{
+    public static function create($feature, $loopName)
+    {
+        return new self('Event loop feature "' . $feature . '" isn\'t supported by the "' . $loopName . '" event loop');
```

Comment:
I'd go for a regular constructor here. The message should probably cover _why_ the feature isn't supported (missing `ext-pcntl`). The `$loopName` should also be the class name, but `"stream_select"` IMO, but I'd argue that it's not needed in the message, because the used implementation will show up in the stack trace.